### DISCUSSION TITLE
new Url(url, base), url passed as a variable cannot get the correct address?

### DISCUSSION THREAD

**Discussion by LJS-Y** [web-infra-dev/rsbuild#3857] [Q&A]
new URL('../assets/images/common/404Bg.jpg', import.meta.url)
正确地址。
const url = '../assets/images/common/404Bg.jpg';
new URL(url, import.meta.url)
错误地址。
为什么？

**Comment by 9aoy** [comment]
Rspack / webpack does not support the use of variables in `new URL`. This is because Rspack cannot statically analyze the syntax.
